packages/teams-js/src/private/interactive.ts

```diff
@@ -0,0 +1,207 @@
+import { sendAndHandleSdkError } from '../internal/communication';
+import { ensureInitialized } from '../internal/internalAPIs';
+import { FrameContexts } from '../public/constants';
+import { errorNotSupportedOnPlatform } from '../public/constants';
+import { runtime } from '../public/runtime';
+
+export namespace interactive {
+  /**
+   * @hidden
+   * Hide from docs
+   * ------
+   * Allowed roles during a meeting.
+   */
+  export enum UserMeetingRole {
+    guest = 'Guest',
+    attendee = 'Attendee',
+    presenter = 'Presenter',
+    organizer = 'Organizer',
+  }
+
+  /**
+   * @hidden
+   * Hide from docs
+   * ------
+   * State of the meetings backing fluid container.
+   */
+  export enum ContainerState {
+    added = 'Added',
+    alreadyExists = 'AlreadyExists',
+    conflict = 'Conflict',
+    notFound = 'NotFound',
+  }
+
+  /**
+   * @hidden
+   * Hide from docs
+   * ------
+   * Information about the meetings backing fluid container.
+   */
+  export interface FluidContainerInfo {
+    containerState: ContainerState;
+    containerId: string | undefined;
+    shouldCreate: boolean;
+    retryAfter: number;
+  }
+
+  /**
+   * @hidden
+   * Hide from docs
+   * ------
+   * The global time servers current timestamp.
+   */
+  export interface NtpTimeInfo {
+    ntpTime: string;
+    ntpTimeInUTC: number;
+  }
+
+  /**
+   * @hidden
+   * Hide from docs
+   * ------
+   * Information about the fluid service to connect to.
+   */
+  export interface FluidTenantInfo {
+    tenantId: string;
+    ordererEndpoint: string;
+    storageEndpoint: string;
+  }
+
+  /**
+   * @hidden
+   * Hide from docs
+   * ------
+   * Returns the Fluid Tenant connection info for user's current context.
+   */
+  export function getFluidTenantInfo(): Promise<FluidTenantInfo> {
+    return new Promise<FluidTenantInfo>(resolve => {
+      ensureInitialized(FrameContexts.meetingStage, FrameContexts.sidePanel);
+      if (!isSupported()) {
```

Comment:
you probably don't want these in your APIs. Teams does not send their 'support' matrix from existing clients, which means this would return false. I think for now it's best to keep this entire module out of the capabilities model.